TipCatDotNet.Api/Services/Payments/ITransactionService.cs

```diff
@@ -16,7 +15,7 @@ public interface ITransactionService
         /// <summary>
         /// Method retrieve succeeded transactions by member
         /// </summary>
-        Task<Result<List<TransactionResponse>>> Get(MemberContext context, PaginationFilter filter, CancellationToken cancellationToken = default);
+        Task<Result<List<TransactionResponse>>> Get(MemberContext context, int skip = 0, int top = 20, CancellationToken cancellationToken = default);
```

Comment:
To enforce code strictness default could be moved to the controller. In addition, that makes `top` value optional